### PR TITLE
📈 Add event log for frontend mode toggle

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: components/dialogs/AuthDialog.vue [hunk 1]
```diff
@@ -725,7 +725,10 @@ export default {
       'loginByCosmosWallet',
     ]),
 
-    toggleFrontendMode,
+    toggleFrontendMode() {
+      logTrackerEvent(this, 'RegFlow', 'ToggleFrontendMode', 'ToggleFrontendModeLegacy)', 1);
```
Comment (williamchong) [inline components/dialogs/AuthDialog.vue:729]: Do we know the exact mode the user is going to here? (`legacy` or `keplr/new`)

**Review by williamchong — CHANGES_REQUESTED** [review]
Can we also set user property on login screen? I am not sure if we are on ua or ga4, but both should have a way to set it
https://developers.google.com/analytics/devguides/collection/ga4/user-properties?technology=websites

File: components/dialogs/AuthDialog.vue (hunk 1 above)
Comment (nwingt) [reply components/dialogs/AuthDialog.vue:729]: Yes, the new UI is using `AuthDialogV2` component, I will add a tracker in it.

Comment (williamchong) [reply components/dialogs/AuthDialog.vue:729]: so `ToggleFrontendModeLegacy` means from legacy to new?
then `ToggleFrontendModeXXX` will be added meaning from new to legacy?

**Review by williamchong — APPROVED** [review]

**Pull request merged**